tests/nataliia_romanchuk/test_exceptions.py

```diff
@@ -0,0 +1,81 @@
+import importlib
+
+import pytest
+
+from nataliia_romanchuk.Classes import Cats
+
+
+def test_attribute_error():
+    cat = Cats('Pushok', 10)
+    print(cat.meow())
+    print(cat.sleep())
```

Comment:
avoid prints since they require visual assertion, 
change it to 
assert

Reply:
fixed